### DISCUSSION TITLE
Xiaomi Mi Box Support

### DISCUSSION THREAD

**Discussion by PRProd** [PRProd/HA-Firemote#60] [Announcements]
## Native support for the Xiaomi Mi Box is here!!!

![image](https://user-images.githubusercontent.com/25486599/229212908-310e9fa0-9a7d-4a11-b7f5-c21cb3c0b7b0.png)

### What's new?
 - Native Support for Xiaomi Mi Box S
 - Most text based app buttons upgraded to actual logos
 - New "Function" commands
 - A few new app shortcut options

**Full Changelog**: https://github.com/PRProd/HA-Firemote/compare/v2.0.4...v2.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/PRProd/HA-Firemote/releases/tag/v2.1.0'>Xiaomi Mi Box Support</a>.</em>

**Comment by alexdelprete** [comment]
@PRProd Doug, first of all let me thank you for this card, I use it on 2 NVIDIA Shields and it is AWESOME. :)

I have one small issue: I read that you supported recently both SmartTubeNext official and Beta versions, but in the list I only see what I think is the official one, since when I press it doesn't launch my beta version of the app. The app id is: `com.liskovsoft.smarttubetv.beta`

![image](https://user-images.githubusercontent.com/7027842/228991772-75f4e8f1-838a-4e2d-95d7-408a9c9a8b4a.png)

Should I see a separate entry in the list for the beta version?

Thanks.

**Reply by PRProd** [reply]
@alexdelprete - I'm glad you're using it and that you like it!!  Thanks for joining the discussions!

I went back and re-read the original request #48 from @MRobi1, and I see what you are talking about.  I missed the beta version request and only did the other two.  Apologies to both of you!  I will re-open #48 and post an update there when the missing shortcut is added.

**Reply by PRProd** [reply]
New release [v2.1.1](https://github.com/PRProd/HA-Firemote/releases/tag/v2.1.1) is now available for download which includes the missing beta button.  Thanks again for pointing it out.

**Comment by alexdelprete** [comment]
Doug, 2 more things:

1. I noticed that Paramount+ is not working. I live in Italy. App ID is: `com.cbs.ca`

2. Could you add support for DAZN? App id: `com.dazn`.

If you want me to open an issue for the requests, or you need more info, please let me know.

Thanks.

**Reply by PRProd** [reply]
@alexdelprete - Thanks for your message!  

The com.cbs.ca app actually exists under "Paramount+ (DE)".  I wrote it that way because someone in Germany noticed the same thing you did (#28).  Since you're in Italy and your app has the same name as the one used in Germany, I am going to change the menu name to "Paramount+ (alt)" instead.  You will see this change in the next release.  In the meantime, you can select "Paramount+ (DE)" to make it work now, and it will continue to work after the next update:
![image](https://user-images.githubusercontent.com/25486599/229174436-5e277632-6067-4447-a729-769b7149f081.png)

For the DAZN app as well as any new app requests, I do prefer that you use the [Issues -> New App Shortcut Request](https://github.com/PRProd/HA-Firemote/issues/new/choose) method.  It looks like #62 was just opened by @schmierlappe for the DAZN app, so It will be handled in that issue thread.  Feel free to keep an eye on #62 to know when it is ready.

**Reply by PRProd** [reply]
New release [v2.1.1](https://github.com/PRProd/HA-Firemote/releases/tag/v2.1.1) is now available for download which includes the more appropriately labeled "Paramount+ (alt)" in the dropdown menu.  Thanks for letting me know!

**Reply by alexdelprete** [reply]
> In the meantime, you can select "Paramount+ (DE)" to make it work now, and it will continue to work after the next update:

Doug, I tried the Paramount+ DE before posting, and as I wrote, it didn't work. 2.1.1 works fine though. :)

**Reply by PRProd** [reply]
@alexdelprete -  Well I'm puzzled... I literally only [changed the label in the dropdown](https://github.com/PRProd/HA-Firemote/compare/v2.1.0...v2.1.1#diff-f1150d2a726ae4a73719bafbaec3ec42c12fcc4e99ecbef093ae81f2966e3a8bR605), I didn't touch the app name or android name.  

![image](https://user-images.githubusercontent.com/25486599/229238522-5b93c0ee-1831-4ce3-93c9-02b4b13f810c.png)


Are you sure you weren't accidentally testing "Paramount+" again thinking that you were testing "Paramount+ (DE)"?

**Comment by alexdelprete** [comment]
If you only changed the label, probably I tested the wrong one. It was very late...:)
